### ISSUE TITLE
[TEST-FAILURE] ClientMapTest.testIssue537

### ISSUE THREAD

**Issue by hasancelik** [hazelcast/hazelcast#6694]
```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.hazelcast.client.map.ClientMapTest.testIssue537(ClientMapTest.java:139)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.6/com.hazelcast$hazelcast-client-legacy/742/testReport/junit/com.hazelcast.client.map/ClientMapTest/testIssue537/

**Comment by hasancelik** [comment]
https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-OracleJDK8/com.hazelcast$hazelcast-client/616/testReport/junit/com.hazelcast.client.map/ClientMapTest/testIssue537/

**Comment by Donnerbart** [comment]
I improved the assert messages on this test a bit: https://github.com/hazelcast/hazelcast/commit/7f4441ec17996d4828739489706e34c73d31bc4d

**Comment by sancar** [comment]
fixed by #6728
